StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheetDataSource.swift

```
@@ -181,6 +181,15 @@ extension CustomerSheetDataSource {
         }
     }
 
+    func setAsDefaultPaymentMethod(paymentMethodId: String, customerID: String) async throws -> STPCustomer? {
+        switch dataSource {
+        case .customerAdapter:
+            return nil
```

Can you add a comment about why we do this? Maybe we should throw an error in this case.

---

I've added a fatal error that explains that customer adapter does not support the set as default feature